Content.Client/Administration/UI/Logs/Entries/AdminLogEntry.xaml.cs

```diff
@@ -0,0 +1,70 @@
+using System.Text.RegularExpressions;
+using Content.Client.Message;
+using Content.Shared.Administration.Logs;
+using Content.Shared.CCVar;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Configuration;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Administration.UI.Logs.Entries;
+
+[GenerateTypedNameReferences]
+public sealed partial class AdminLogEntry : BoxContainer
+{
+    private readonly IConfigurationManager _cfgManager;
+    private int ClipLength;
+    public SharedAdminLog Log { get; }
+
+    private readonly string _rawMessage;
+
+    public AdminLogEntry(ref SharedAdminLog log)
+    {
+        _cfgManager = IoCManager.Resolve<IConfigurationManager>();
+
+        RobustXamlLoader.Load(this);
+
+        Log = log;
+
+        _rawMessage = $"{log.Date:HH:mm:ss}: {log.Message}";
+        Message.SetMessage(_rawMessage);
+
+        DetailsHeading.OnToggled += DetailsToggled;
+    }
+
+    /// <summary>
+    /// Sets text to be highlighted from a search result, and renders rich text, or removes all rich text markup.
+    /// </summary>
+    public void RenderResults(Regex highlightRegex, bool renderRichText, bool removeMarkup)
+    {
+        var color =_cfgManager.GetCVar(CCVars.AdminLogsHighlightColor);
```

Comment:
```suggestion
        var color = _cfgManager.GetCVar(CCVars.AdminLogsHighlightColor);
```

also I recommend `cvar ui.admin_logs_highlight_color "red][font=\"AnimalSilence\""` (This isn't an issue; I just think it's funny.)

Reply:
this is all client-side so I don't really care about that lol